public/app/features/dashboard/components/SubMenu/SubMenu.tsx

```diff
@@ -67,6 +70,12 @@ const mapStateToProps: MapStateToProps<ConnectedProps, OwnProps, StoreState> = (
   };
 };
 
+const styles = css`
+  display: flex;
+  flex-wrap: wrap;
+  display: contents;
+`;
+
```

Comment:
Oops, I didn't realize this was a class component 🙈

Reply:
Yeah, doing it this way will only make it being called once, right?